study.md

```diff
@@ -115,5 +115,5 @@ alternative. How might you justify this choice of best alternative gem to a
 senior engineer on your team? Explain this below and submit a pull request.
 
 ```md
-<!-- your answer here -->
+I did the readings but really had a hard time trying to research gems.
```

Comment:
The website provided https://rubygems.org/gems has tons of cool gems you can read about. Its good to check out the github repo that it is stored in too. You can see how often it is updated and how many people find it useful.